File: estudiante-service/src/main/java/com/example/estudianteservice/services/EstudianteService.java
package com.example.estudianteservice.services;

import com.example.estudianteservice.entities.EstudianteEntity;
import com.example.estudianteservice.repositories.EstudianteRepository;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import java.util.ArrayList;
@Service
public class EstudianteService {
    @Autowired
    EstudianteRepository estudianteRepository;
    public ArrayList<EstudianteEntity> obtenerEstudiantes(){
        return (ArrayList<EstudianteEntity>) estudianteRepository.findAll();
    }

    public void guardarEstudiante(EstudianteEntity estudiante){
        estudianteRepository.save(estudiante);
    }
    public void borrarTodo(){
        estudianteRepository.deleteAll();
    }
}

File: frontend/src/services/EstudianteService.js
import axios from 'axios';
const URL = "http://localhost:8083/estudiante";

class EstudianteService{
    obtenerEstudiantes(){
        return axios.get(URL);
    }
    borrarTodo(){
        return axios.post(URL +"/borrarEstudiantes")
    }
    crearEstudiante(estudiante){
        return axios.post(URL +"/crearEstudiante",estudiante)
    }
}
export default new EstudianteService()